starter-code/public/scripts/models/repo.js

```diff
@@ -11,8 +11,21 @@ var app = app || {};
     //       Remember that the callback function we'll want to call relies on repos.all
     //       being an array with a bunch of repo objects in it, so you'll need to
     //       populate it with the response from Github before you call the callback.
-
-  };
+    $.ajax({
+      url: `http://api.github.com/users/repos`,
```

Comment:
This needs to be: https://api.github.com/users/”username”/repos or https://api.github.com/user/repos.